mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -710,6 +734,9 @@
         <EditorCategories value="AbilityorEffectType:Units"/>
         <Duration value="0.2"/>
         <AINotifyEffect value="SeekerMissileDamage"/>
+        <DamageResponse>
+            <ModifyAmount value="2"/>
+        </DamageResponse>
```

Comment:
Bogus change with the chance being 0.

This would double the amount of damage received, if it were enabled with chance set to 1

=> should be removed